backend/src/test/java/com/github/chuettenrauch/mixifyapi/integration/auth/service/AuthServiceTest.java

```diff
@@ -0,0 +1,33 @@
+package com.github.chuettenrauch.mixifyapi.integration.auth.service;
+
+import com.github.chuettenrauch.mixifyapi.auth.service.AuthService;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.test.context.support.WithMockUser;
+import org.springframework.test.context.TestPropertySource;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@SpringBootTest
+@TestPropertySource(properties = "spring.autoconfigure.exclude=de.flapdoodle.embed.mongo.spring.autoconfigure.EmbeddedMongoAutoConfiguration")
```

Comment:
What does that annotation do?

Reply:
Die annotation verhindert, dass für diesen Test eine Flapdoodle Mongo hochgefahren wird. 
Das spart Testausführungszeit, wenn man Integration Tests hat, wo keine DB gebraucht wird.